barretenberg/barretenberg.code-workspace

```diff
@@ -152,8 +152,7 @@
         },
         "cmake.configureArgs": [
             "--preset clang16",
-            "-G Ninja",
-            "-g"
```

Comment:
This flag was preventing my CMakeTools from building... I guess removing it didn't give you problems?